.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Bug report
+about: Create a report describing unexpected or incorrect behavior in astropy.
+---
+
+<!-- This comments are hidden when you submit the issue,
+so you do not need to remove them! -->
+
+<!-- Please be sure to check out our contributing guidelines,
+https://github.com/astropy/astropy/blob/master/CONTRIBUTING.md .
+Please be sure to check out our code of conduct,
+https://github.com/astropy/astropy/blob/master/CODE_OF_CONDUCT.md . -->
+
+<!-- Please have a search on our GitHub repository to see if a similar
+issue has already been posted.
+If a similar issue is closed, have a quick look to see if you are satisfied
+by the resolution.
+If not please go ahead and open an issue! -->
+
+<!-- Please check that the development version still produces the same bug.
+You can install development version with
+pip install git+https://github.com/astropy/astropy
+command. -->
+
+### Description
+<!-- Provide a general description of the bug. -->
+
+### Expected behavior
+<!-- What did you expect to happen. -->
+
+### Actual behavior
+<!-- What actually happened. -->
+<!-- Was the output confusing or poorly described? -->
+
+### Steps to Reproduce
+<!-- Ideally a code example could be provided so we can run it ourselves. -->
+<!-- If you are pasting code, use tripe backticks (```) around
+your code snippet. -->
+<!-- If necessary, sanitize your screen output to be pasted so you do not
+reveal secrets like tokens and passwords. -->
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+```python
+# Put your Python code snippet here.
+```
+
+### System Details
+<!-- Even if you do not think this is necessary, it is still nice to have. -->
+
+<!-- import astropy; print(astropy.__version__) -->
```

Comment:
I like at sklearn land that they have a copy paste-able block for this:
```
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import astropy; print("Astropy", astropy.__version__)
```

Reply:
Done.